pkg/apis/compute/v1alpha1/register.go

```diff
@@ -30,11 +30,13 @@ import (
 )
 
 const (
-	Group                  = "compute.crossplane.io"
-	Version                = "v1alpha1"
-	APIVersion             = Group + "/" + Version
-	KubernetesInstanceKind = "kubernetesclusters"
-	WorkloadKind           = "workloads"
+	Group                            = "compute.crossplane.io"
+	Version                          = "v1alpha1"
+	APIVersion                       = Group + "/" + Version
+	KubernetesInstanceKind           = "kubernetesclusters"
```

Comment:
we've been using the singular form in most other types, we should probably keep it consistent

Reply:
:+1: